File: go/hello/hello_test.go
// M22 seed Go test; consumer of dx_go_test.
package hello

import "testing"

func TestHello(t *testing.T) {
	t.Parallel()
	if got, want := Hello("world"), "hello world"; got != want {
		t.Fatalf("Hello(world) = %q, want %q", got, want)
	}
}

File: scala/hello/HelloTest.scala
// M23 seed Scala test; consumer of dx_scala_test (ScalaTest).
package hello

import org.scalatest.flatspec.AnyFlatSpec

class HelloTest extends AnyFlatSpec {
  "Hello.hello" should "greet by name" in {
    assert(Hello.hello("world") == "hello world")
  }
}
